### PR TITLE
minior improve serve_connect

### PR DESCRIPTION
**Pull request by zirain**

### CODE REVIEW

**Review by hzxuzhonghu — APPROVED** [review]

File: src/proxy/inbound.rs [hunk 1]
```diff
@@ -147,33 +147,39 @@ impl Inbound {
...
                 }
 
                 let addr: SocketAddr = addr.unwrap();
-                *res.status_mut() =
-                    match Self::handle_inbound(InboundConnect::Hbone(req), addr).await {
-                        Ok(_) => StatusCode::OK,
-                        Err(_) => StatusCode::SERVICE_UNAVAILABLE,
-                    };
-                Ok(res)
+                // let mut res = Response::new(Body::empty());
```
Comment (PiotrSikora) [inline src/proxy/inbound.rs:164]: Nit: remove this comment.

**Review by PiotrSikora — APPROVED** [review]
LGTM.

File: src/proxy/inbound.rs (hunk 1 above)
Comment (zirain) [reply src/proxy/inbound.rs]: thanks, updated.

**Review by howardjohn — APPROVED** [review]
thanks

**Pull request merged**